tests/e2e/stablehlo_ops/batch_norm_inference.mlir

```diff
@@ -0,0 +1,13 @@
+func.func @batchnorm_inference_4x2() {
+  %x = util.unfoldable_constant dense<[[1.0, 2.0, 3.0, 4.0],[5.0, 6.0, 7.0, 8.0]]> : tensor<2x4xf32>
+  %mean = util.unfoldable_constant dense<[1.0, 1.0, 1.0, 1.0]> : tensor<4xf32>
+  %var = util.unfoldable_constant dense<[2.0, 2.0, 2.0, 2.0]> : tensor<4xf32>
+  %offset = util.unfoldable_constant dense<[1.0, 1.0, 1.0, 1.0]> : tensor<4xf32>
+  %scale = util.unfoldable_constant dense<[1.0, 1.0, 1.0, 1.0]> : tensor<4xf32>
+  %result = "stablehlo.batch_norm_inference"(%x, %mean, %var, %offset, %scale) {epsilon = 1.000000e-03 : f32, feature_index = 1 : i64} : (tensor<2x4xf32>, tensor<4xf32>, tensor<4xf32>, tensor<4xf32>, tensor<4xf32>) -> tensor<2x4xf32>
+  // TODO(gcmn): This should probably be a fuzzier check with round values.
```

Comment:
Wait, why is there a TODO for me here?

Reply:
It's copied from the MHLO version of the same test.

Reply:
Oh lol that TODO is old: https://github.com/openxla/iree/pull/11540 changed it to point directly at me and the original is from https://github.com/openxla/iree/commit/eb800738ce! Fine to leave it